.github/workflows/pages.yml

```diff
@@ -5,6 +5,7 @@ on:
   # Runs on pushes targeting the default branch
   push:
     branches: ["main"]
```

Comment:
Just put `main` below like `- main`. I just think it looks nicer.

Reply:
Done